src/test/java/utilities/DriverClass.java

```diff
@@ -23,6 +23,11 @@ public class DriverClass {
     public static WebDriver getDriver() {
 
         if (threadDriver.get() == null || !threadDriver.get().toString().contains(threadDriverName.get())) {
+
+            if (threadDriverName.get()==null){
+                threadDriverName.set("chrome");
+            }
```

Comment:
Kod formatlama yapılabilir, bu sayede kodun formatı düzgün olduğunda daha okunaklı hâle getiriliyor. `threadDriverName.get()==null` yerine `threadDriverName.get() == null` şeklinde olması gerekiyor. Bu durumda kısa yollar ile formatlama veya commit atmadan önce formatlama yapılabiliyor. 

Kaynak olarak https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html dokümanı incelebilir.